src/appleseed/renderer/kernel/rendering/final/adaptivepixelrenderer.cpp

```diff
@@ -306,6 +306,23 @@ namespace
             return m_params.m_max_samples;
         }
 
+        void print_settings() const override
+        {
+            RENDERER_LOG_INFO(
+                        "adaptive pixel renderer settings:\n"
+                        "  sampling mode                 %s\n"
+                        "  min samples                   %zu\n"
```

Comment:
For maximum portability, use special format strings defined in `foundation/platform/types.h`, e.g. `FMT_SIZE_T` for `std::size_t` values.